Allow collapsing the traceability information for item

JasperCraeghs opened the pull request:
If an _item_ has any rendered _relationships_ or _attributes_, a button will be added on the right hand side that allows you to hide this info. 
Hiding (collapsing) all links on page load can be enabled by setting `traceability_collapse_links` to `True` in `conf.py`. The links are shown (uncollapsed) by default.

<img src="https://user-images.githubusercontent.com/28319872/61730280-5928b980-ad79-11e9-81bc-95279060beeb.png" width="500" >

Closes #92

JasperCraeghs commented:
What should the color of the arrowhead button be? It is currently the same as the color of attribute/relationship names, e.g. depends_on. Do you want the collapsing to be animated or instant?

JasperCraeghs commented:
Do we want this feature enabled by default?

Letme commented:
Adding @bavovanachte to provide feedback on open questions.

By default I would keep it open, as it will be easier for beginners to see which links are rendered towards the iten.

JasperCraeghs commented:
Oh, like that. I thought the configurability was to enable or disable the additions of the buttons. I suppose that now that the _Hide links_ text is replaced by the arrow down button, the feature is less obstrusive. I'll change the configurability aspect.

bavovanachte commented:
Thanks for taking this up!

- Color looks good based on the screenshots you posted
- I could personally live without the animation, but it's a nice touch
- As for configurability: I would expect a configuration that allows selecting between hiding and showing the links for all items (not sure if that's what you mean by "enabling the feature"). Having a default where they are all visible is fine for me.

JasperCraeghs commented:
Thanks for the feedback. I've changed the configurability aspect and added an animation. I also improved the button's position to prevent it from overlapping with an admonition  when the item's content is empty.

Letme commented on mlx/assets/traceability.js:
```diff
@@ -1,4 +1,45 @@
+                }
+            });
+            admonition.after(arrow);
+
```
Blank line here not needed(wanted).

JasperCraeghs replied:
7f2ae587562775be458242e0060b8aef269190e8

Letme commented on mlx/assets/traceability.js:
```diff
@@ -1,4 +1,45 @@
+        if (relations.children().length > 0) {
+            if (admonition.parent().hasClass('collapse')) {
+                // collapse relations and attributes list for each item on page load
+                relations.toggle();
```
Toggle here but not in else - Did you try the close/open/close/open cycle?

JasperCraeghs replied:
This `toggle()` should only happen once (on page load) and only when the config variable has been set to `True` (which adds the 'collapse' class in Python code). It collapses the list of links without animation.

I have tried all the cycles and both configuration options. It works as intended. You can give it a go yourself to see if you want a change to the positioning, button scale, animation, ... :)
